SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt

```diff
@@ -336,7 +336,8 @@ fun CryptoAssetItem(crypto: SymbolPriceBankModel,
                     index:Int, selectedIndex:Int,
```

Comment:
Not sure if you have a linter for this, but I think it would be nice to have a space after the `:` 
```suggestion
                    index: Int, selectedIndex: Int,
```